source/_components/sensor.tautulli.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: "Tautulli activity sensor"
+description: "Instructions on how to set up Tautulli sensors in Home Assistant."
+date: 2018-10-27 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tautulli.png
+ha_category: Sensor
+ha_release: 0.82
+ha_iot_class: "Local Polling"
+---
+
+The `Tautulli` sensor platform will monitor activity on a given [Tautulli Server][tautulli]. It will create a sensor that shows the number of currently active streams as the state. If you click the sensor for more details it will show you more statistics, the folowing stats are aviable by default:
```

Comment:
```suggestion
The `Tautulli` sensor platform will monitor activity on a given [Tautulli Server][tautulli]. It will create a sensor that shows the number of currently active streams as the state. If you click the sensor for more details it will show you more statistics, the following stats are available by default:
```